src/Rings/mpoly-affine-algebras.jl

```diff
@@ -19,20 +19,54 @@ julia> dim(A)
 1
 ```
 """
-function dim(A::MPolyQuoRing)
-  return dim(modulus(A))
+dim(A::Union{MPolyQuoRing, zzModRing, ZZModRing}) = krull_dim(A)
+
+@doc raw"""
+    krull_dim(A::MPolyQuoRing)
+
+Return the Krull dimension of `A`.
+
+# Examples
+```jldoctest
+julia> R, (x, y, z) = polynomial_ring(QQ, [:x, :y, :z]);
+
+julia> A, _ = quo(R, ideal(R, [y-x^2, z-x^3]));
+
+julia> krull_dim(A)
+1
+```
+"""
+function krull_dim(A::MPolyQuoRing)
+  return krull_dim(modulus(A))
 end
 
-function dim(A::zzModRing)
+function krull_dim(A::zzModRing)
```

Comment:
```suggestion
function krull_dim(A::Union{zzModRing,ZZModRing})
```
Of course this is type piracy. These methods really should be defined in `Nemo.jl`.